test/lib/file_storage/local_file_test.rb

```diff
@@ -12,8 +12,14 @@ def storage(**)
   end
 
   test 'can save a file' do
-    FileStorage::LocalFile.new.save_file 'abc', 'xyz'
-    FileStorage::LocalFile.new(path: storage_path).save_file 'abc', 'xyz'
+    temp_storage = FileStorage::LocalFile.new
+    temp_storage.save_file 'abc', 'xyz'
+    assert_equal 'xyz', File.read(File.join(temp_storage.directory, 'abc'))
+  end
+
+  test 'can save a file in a configured location' do
+    storage(path: storage_path).save_file 'abc', 'xyz'
+    assert_equal 'xyz', File.read("#{storage_path}/abc")
```

Comment:
Not actually required, but Rails started raising warnings about tests without assertions. These are probably better tests now anyway (I guess the warning did its job!).